internal/locate/region_request.go

```diff
@@ -1487,6 +1477,11 @@ func (s *RegionRequestSender) SendReqCtx(
 			// TODO: Change the returned error to something like "region missing in cache",
 			// and handle this error like EpochNotMatch, which means to re-split the request and retry.
 			s.logSendReqError(bo, "throwing pseudo region error due to no replica available", regionID, retryTimes, req, totalErrors)
+			if s.replicaSelector != nil {
```

Comment:
Are the L1480 and L1514 strategy change that we should also backoff when there is no candidate or before retry, compared with the orginal logic?

Reply:
The original logic is, whether there is a candidate or not, do backoff first when an region error is encountered.

compared with the original logic, you can see the `TestReplicaReadAccessPathByCase` change, some test-case's `backoffCnt` and `backoffDetail` are changed.